Fixed warning message for when Filter is on EndPath

Dr15Jones opened the pull request:
At some time in the past when the fillWorkers function was changed the warning message stopped printing the name of the path and instead printed the name of the module twice.

Dr15Jones committed:
Fixed warning message for when Filter is on EndPath

At some time in the past when the fillWorkers function was changed the warning message stopped printing the name of the path and instead printed the name of the module twice.

cmsbuild commented:
A new Pull Request was created by  @Dr15Jones (Chris Jones) for CMSSW_8_1_X.

It involves the following packages:

FWCore/Framework


@cmsbuild, @smuzaffar, @Dr15Jones, @davidlange6 can you please review it and eventually sign? Thanks.
@Martin-Grunewald, @wddgit, @wmtan this is something you requested to watch as well.
@slava77, @smuzaffar you are the release manager for this.

cms-bot commands are list here cms-sw/cmssw#13028

Dr15Jones commented:
please test

Dr15Jones commented:
+1

cmsbuild commented:
The tests are being triggered in jenkins.

cmsbuild commented:
This pull request is fully signed and it will be integrated in one of the next CMSSW_8_1_X IBs after it passes the integration tests. This pull request requires discussion in the ORP meeting before it's merged. @slava77, @davidlange6, @smuzaffar

Dr15Jones committed:
Fix warning when path contains unknown module

The warning message was printing the label for the module twice instead of printing both the module label and the path name.

cmsbuild commented:
Pull request #15832 was updated. @cmsbuild, @smuzaffar, @Dr15Jones, @davidlange6 can you please check and sign again.

Dr15Jones commented:
please test

Dr15Jones commented:
+1

cmsbuild commented:
The tests are being triggered in jenkins.

cmsbuild commented:
This pull request is fully signed and it will be integrated in one of the next CMSSW_8_1_X IBs after it passes the integration tests. This pull request requires discussion in the ORP meeting before it's merged. @slava77, @davidlange6, @smuzaffar

cmsbuild commented:
+1
Tested at: 73848729fe5db01f4dc948727944a03f338c1a75
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-15832/15111/summary.html

cmsbuild commented:
+1
Tested at: e251cfe53920f8ded79ff6ee996b919505ccdcc4
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-15832/15112/summary.html

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-15832/15111/summary.html

@slava77 comparisons for the following workflows were not done due to missing matrix map:
   * 20424.0_TTbar_13+TTbar_13TeV_TuneCUETP8M1_2023D2_GenSimFull+DigiFull_2023D2+RecoFullGlobal_2023D2+HARVESTFullGlobal_2023D2
